rfcs/web_extensions.md

```diff
@@ -0,0 +1,199 @@
+# RFC #219: Add support for WebExtensions in WPT
+
+## Summary
+
+The WebExtensions API extends the capabilities of the browser. Adding support for WebExtensions in
+the web-platform-tests will increase interoperability and will help drive the standardization of
+this API.
+
+This RFC proposes adding a new `testharness.js` wrapper type, `.extension.js`, to handle testing this
+API, in addition to using `testdriver.js` to load and unload extensions. A new `feature=extensions`
+was also introduced to support enabling `BiDi` specifcally for user agents who use it for the
```

Comment:
Just as a style note, these are usually written in the present or future tense rather than the past, as a description of what the proposal is and what changes it will make.